sidebars.js

```diff
@@ -68,5 +68,6 @@ module.exports = {
       ]
     }],
     GSoC: [ 'gsoc/gsoc-ideas' ],
+    "Brand Assets": ["branding/logos", "branding/color-palette", "branding/typography", "branding/patterns-illustrations"],
```

Comment:
Change the heading from "Brand Assets" to Branding. Move the section above GSoC

Reply:
Okay sure 💯